### DISCUSSION TITLE
お知らせを全体とユーザー単位で分けてほしい

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#5] [要望・アイデア・不具合]
右上のベルに表示されるお知らせを、全体（Qiita からのお知らせ）とユーザー単位（コメントや被フォローなど）で分けてほしいです。
全体のお知らせはベルにバッジが付かないとより嬉しいです。

**Comment by kskwtnk** [comment]
@uhooi 
ご意見ありがとうございます！

通知機能はたしかに煩雑になってきているのですが、

> 全体（Qiita からのお知らせ）とユーザー単位（コメントや被フォローなど）で分けてほしい

uhooiさんが上記のように感じるのは、どういうモチベーションからでしょう？
背景までお聞きできた方がより良い開発ができると思うので、是非教えてください！

**Reply by uhooi** [reply]
失礼な言い方になったら申し訳ないですが、全体のお知らせよりユーザー単位のお知らせのみを見たいのです…。
アプリのプッシュ通知にもいえますが、全体のお知らせが多いとユーザーはお知らせ自体を見なくなってしまいます。
今のところ全体のお知らせは多くないですが、私は読み飛ばしていることがほとんどです…。

**Reply by kskwtnk** [reply]
@uhooi 

いえいえ、失礼なんてとんでもないです、ありがとうございます！

背景を把握できましたので、こちらも検討していきたいと思います！
ちなみに……社内での認識を共有しますと「通知って今の状態ではあまり便利ではないよね」という議論は出るものの、Issueとしてちゃんと進められていない、みたいな状態です 🙇

**Comment by blono** [comment]
こういう機能を実装するときに、ユーザが得たい通知をチェックボックで選べるようにするのか、そうではなく固定するのかまで、実装の幅がありそうですね。
ユーザからの通知、あるいは Qiita からの通知といっても、その内容は種々様々でしょうし。

**Reply by kskwtnk** [reply]
@blono 

> ユーザからの通知、あるいは Qiita からの通知といっても、その内容は種々様々でしょうし。

そうなんです、現状でもマイページの設定に結構たくさんの通知がありますので、それぞれの扱いも含めて色々考える必要がありそうです

**Comment by tsuzuki-takaaki** [comment]
@uhooi
報告が遅くなってしまい、申し訳ございません。

いただいたご要望と完全に一致する形ではありませんが、
整理・管理をしやすくするために、通知をカテゴリーごとに確認できるようにしました！
https://qiita.com/release-notes/2024-09-11

ご活用いただけると幸いです🙇‍♂️

**Reply by uhooi** [reply]
ありがとうございます！！✨
通知一覧を一切開かないので気づきませんでした…。
